openedx/core/djangoapps/ace_common/templatetags/ace.py

```diff
@@ -11,6 +11,16 @@
 register = template.Library()  # pylint: disable=invalid-name
 
 
+@register.simple_tag(takes_context=True)
+def get_action_links(context, channel, omit_unsubscribe_link=False):
+    """
+    Retrieve the action links for the channel and pass the omit_unsubscribe_link argument
+    """
+    if getattr(channel, 'action_links', None):
+        return channel.action_links(omit_unsubscribe_link=context.get('omit_unsubscribe_link'))
+    return []
```

Comment:
What is the use of this? Are we not able to just call it from edx-ace directly? If so, why not?

Reply:
if the question is why i added the template tag, i believe you can't pass arguments to function calls in django templates, you need a template tag for that purpose